.eslintrc.js

```diff
@@ -31,16 +31,24 @@ module.exports = {
 		'prefer-const': 1,
 		'no-const-assign': 2
 	},
-	'globals': {
-		'after': true,
-		'afterEach': true,
-		'before': true,
-		'beforeEach': true,
-		'describe': true,
-		'fetch': true,
-		'it': true,
-		'spawnTest': true,
-		'spawnTestWithTempdir': true,
-		'describeWithPackages': true
-	}
+	overrides: [
+		{
+			files: ['**/*.test.js', 'test/testhelper.js'],
+			env: {
+				mocha: true,
+			},
+			'globals': {
+				'after': true,
+				'afterEach': true,
+				'before': true,
+				'beforeEach': true,
+				'describe': true,
+				'fetch': true,
+				'it': true,
```

Comment:
These are included by defined the `env` as `mocha`.
```suggestion
				'fetch': true,
```